ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -404,6 +404,14 @@ tests:
     cluster_profile: azure4
     workflow: openshift-e2e-azure
   timeout: 6h0m0s
+- as: e2e-azure-ovn-techpreview
+  run_if_changed: azure
```

```suggestion
  optional: true
```
We don't want this job being automatically triggered.

---

also `always_run: false`